invenio/modules/knowledge/templates/knowledge/list.html

```diff
@@ -25,13 +25,13 @@
     </li>
     {% if admin_view.can_create %}
     <li>
-        <a href="{{ get_url('.create_view', url=return_url, kbtype='t') }}" title="{{ _('Create new record "Taxonomy"') }}">{{ _('Create KB "Taxonomy"') }}</a>
+        <a href="{{ get_url('.create_view', url=return_url, kbtype='t') }}" title="{{ _('Create new record &#34;Taxonomy&#34;') }}">{{ _('Create KB "Taxonomy"') }}</a>
```

Comment:
I'm not sure if it's smart to have HTML entities in translation strings. Perhaps better write a title without need of the quotes or alternatively, change from ``title=""`` to ``title=''``